meta-mender-core/classes/mender-helpers.bbclass

```diff
@@ -19,6 +22,9 @@ get_uboot_device_from_device() {
         /dev/mmcblk[0-9])
             echo ${1#/dev/mmcblk}
             ;;
+        ubi[0-9])
+            echo ubi
```

Comment:
Glad you caught this, I wanted to drop this change. Because it is incorrect.

If we look at my kernel command line:
``` 
Kernel command line: ubi.mtd=ubi root=ubi0_0 rootfstype=ubifs ubi.fm_autoconvert=1
```

and in U-boot:
```
ubiargs=ubi.mtd=${mender_uboot_dev} root=${mender_kernel_root} rootfstype=ubifs ubi.fm_autoconvert=
```

As you can see I have used `mender_uboot_dev` and that is why I modified helper class. But what I need to assign to `ubi.mtd=` is the `name` or `number` of the MTD part where UBI is present. 

In my case it is `3` or `ubi` taken from below configuration
```
Colibri VFxx # mtdparts

device nand0 <vf610_nfc>, # parts = 4
 #: name                size            offset          mask_flags
 0: vf-bcb              0x00020000      0x00000000      1
 1: u-boot              0x00160000      0x00020000      1
 2: u-boot-env          0x00100000      0x00180000      0
 3: ubi                 0x1fd80000      0x00280000      0
```

It is probably bad to enforce the `ubi` name which I do with how it works currently in `mender-helper`.

All of this was building up to a question. Is it correct of me to re-use the `mender_uboot_dev` for this use-case, or should I create a new configuration option?

If it is OK to re-use the `mender_uboot_dev` I will probably drop above change and require it to be configured with `MENDER_UBOOT_STORAGE_DEVICE` only.